portfolio/src/main/webapp/style.css

```diff
@@ -55,20 +62,61 @@ body {
     width: auto;
 }
 
+/*************** loader ***************/
+.loader {
+  border: 16px solid #f3f3f3; /* Light grey */
+  border-top: 16px solid #3498db; /* Blue */
+  border-radius: 50%;
+  width: 120px;
+  height: 120px;
+  animation: spin 2s linear infinite;
+  
+  position: absolute;
+  top: 40%;
+  left: 50%;
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
+
+/* citation: https://www.w3schools.com/howto/howto_css_loader.asp */
```

Comment:
Remove the citation. Unless the code is complicated and you are trying to link back to the tutorial for future developers, they're not needed.